pandoc.bzl

```diff
@@ -52,29 +52,48 @@ PANDOC_EXTENSIONS = {
 def _pandoc_impl(ctx):
     toolchain = ctx.toolchains["@bazel_pandoc//:pandoc_toolchain_type"]
     cli_args = []
+    all_outputs = [ctx.outputs.output]
+    all_data_inputs = []
+    all_data_inputs.extend(ctx.attr.data)
     cli_args.extend(ctx.attr.options)
     if ctx.attr.from_format:
         cli_args.extend(["--from", ctx.attr.from_format])
     if ctx.attr.to_format:
         cli_args.extend(["--to", ctx.attr.to_format])
+    if ctx.attr.css:
+        all_data_inputs.extend([ctx.attr.css])
+        cli_args.extend(["-c", ctx.file.css.path])
```

Comment:
Isn't it better to use ctx.expand_location instead of forcing an option?

https://docs.bazel.build/versions/master/skylark/lib/ctx.html#expand_location

Reply:
I don't get it, the CSS is an input. Can you post the code you would prefer ?

Reply:
if ctx.attr.data:
        # Expand location from the data argument.
        expanded_options = [ctx.expand_location(option, targets = ctx.attr.data) for option in ctx.attr.options]
        cli_args.extend(expanded_options)
        data_args = ctx.files.data
    else:
        # No data to expand location with.
        cli_args.extend(ctx.attr.options)
        data_args = []

Reply:
In that way, the user can add $(location //my/file) arguments.

Note that the data argument has a very specific meaning in Bazel, and the user does not expect this files to be copied. Some files needs to be copied, some not, let the data argument be true to the Bazel definition of data, and add another argument like "data_copy" that copies the files to the current location instead of the self_contained flag.

For a complete pandoc implementation, some inputs will need to be copied and some not

Reply:
Here is an example of a custom python filter:
```
pandoc(
    name = "MyName",
    src = "MyName.md",
    from_format = "markdown",
    to_format = "html",
    options = [
        "--filter=$(location :pandoc_custom_filter.py)",
    ],
    data = [
        ":pandoc_custom_filter.py",
    ],
)
```